packages/web-components/@sfx/sayt/src/sayt.ts

```diff
@@ -211,6 +211,8 @@ export default class Sayt extends LitElement {
    */
   hideSayt(): void {
     this.visible = false;
+    this.debouncedRequestSaytProducts.clear();
+    this.visible = false;
```

Comment:
Why is this necessary? If it's a timing issue, would it be sufficient to clear first, then set visible?